bin/release.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+rake db: migrate
+rake feature_flags: initialize
```

Comment:
not related to this PR but I usually recommend running all rails/rake commands prepended with `bundle exec`

https://thoughtbot.com/blog/but-i-dont-want-to-bundle-exec